Week2/Week2/StoredMemo.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  StoredMemo.swift
+//  Week2
+//
+//  Created by 정종인 on 2022/04/19.
+//
+
+import Foundation
+
+class StoredMemo {
+    static let shared = StoredMemo()
+    
+    private var memos: [Memo] = []
+    
+    private init() { }
+    
+    public func addMemo(memo: Memo) {
+        memos.append(memo)
+    }
+    
+    public func getMemo(at index: Int) -> Memo {
+        return memos[index]
+    }
+    
+    public func getMemosCount() -> Int {
```

Comment:
get prefix를  스위프트 가이드라인에서는 잘 안쓰더라고요
좀 더 스위프트스러운(?) 입니다
```suggestion
    public var count:  Int {
        return memos.count
    }
```